aiosmtpd/controller.py

```diff
@@ -8,13 +8,16 @@
 
 @public
 class Controller:
-    def __init__(self, handler, loop=None, hostname='::0', port=8025):
+    def __init__(self, handler, loop=None, hostname='::0', port=8025,
+                 thread_timeout=5):
```

Comment:
I agree it would be good to be able to set this in code, not just environment variables.

Actually, I'd say it should only be settable via code. If someone wants to read an environment variable and pass that in as a parameter, they can. But if it's only controlable as an environment variable, you're out of luck if you want to pass in a value from your own config file or some other place.

And aren't there embedded environments where there's no such thing as an environment variable, anyway?